jenkins/test-history/gen_history

```diff
@@ -28,16 +27,16 @@ readonly jsonpath="gs://${bucket}/logs/$(date +%F).json"
 # generated.
 gsutil -q cp "${jsonpath}" "tests.json" || true
 
-# Create JSON report
-time python gen_json.py \
-  --buckets=buckets.json \
-  "--match=^kubernetes|kubernetes-build|kubelet-gce-e2e-ci"
-
-# Create static HTML reports out of the JSON
-python gen_html.py \
-    --output-dir=static \
-    --input=tests.json \
-    --buckets=buckets.json
+docker run --rm -v "$(pwd):/test-history" -w="/test-history" python:2.7 bash -c "\
```

Comment:
pass the `-i` flag so any errors are visible instead of silently buried in a docker log